packages/react/src/BlockNoteView.tsx

```diff
@@ -1,33 +1,66 @@
 import { BlockNoteEditor, BlockSchema } from "@blocknote/core";
-import { MantineProvider } from "@mantine/core";
+import { createStyles, MantineProvider } from "@mantine/core";
 import { EditorContent } from "@tiptap/react";
 import { HTMLAttributes, ReactNode } from "react";
-import { getBlockNoteTheme } from "./BlockNoteTheme";
+import { blockNoteToMantineTheme, Theme } from "./BlockNoteTheme";
 import { FormattingToolbarPositioner } from "./FormattingToolbar/components/FormattingToolbarPositioner";
 import { HyperlinkToolbarPositioner } from "./HyperlinkToolbar/components/HyperlinkToolbarPositioner";
 import { SideMenuPositioner } from "./SideMenu/components/SideMenuPositioner";
 import { SlashMenuPositioner } from "./SlashMenu/components/SlashMenuPositioner";
 
+function UnThemedBlockNoteView<BSchema extends BlockSchema>(
+  props: {
+    editor: BlockNoteEditor<BSchema>;
+    children?: ReactNode;
+  } & HTMLAttributes<HTMLDivElement>
+) {
+  const { classes } = createStyles({ root: {} })(undefined, {
+    name: "Editor",
+  });
+
+  const { editor, children, className, ...rest } = props;
+
+  return (
+    <EditorContent
+      editor={props.editor?._tiptapEditor}
+      className={
+        props.className ? `${classes.root} ${props.className}` : classes.root
+      }
+      {...rest}>
+      {props.children || (
+        <>
+          <FormattingToolbarPositioner editor={props.editor} />
+          <HyperlinkToolbarPositioner editor={props.editor} />
+          <SlashMenuPositioner editor={props.editor} />
+          <SideMenuPositioner editor={props.editor} />
+        </>
+      )}
+    </EditorContent>
+  );
+}
+
 export function BlockNoteView<BSchema extends BlockSchema>(
   props: {
     editor: BlockNoteEditor<BSchema>;
+    useDarkTheme?: boolean;
+    customTheme?: Partial<Theme>;
     children?: ReactNode;
   } & HTMLAttributes<HTMLDivElement>
 ) {
-  const { editor, children, ...rest } = props;
+  const { classes } = createStyles({ root: {} })(undefined, {
```

Comment:
I think you can also pass the theme to `createStyles`? In that case the extra `UnThemedBlockNoteView` is not necessary

Reply:
Hmm don't think so - instead of an object containing styles (in this case `{ root: {} }`), you can provide a function which takes a theme and returns an object containing styles. But that theme param is still taken from the Context, I don't think there's a way to actually pass a theme in yourself:

```ts
export declare function createStyles<
  Key extends string = string, 
  Params = void, 
  Input extends Record<Key, CSSObject> = Record<Key, CSSObject>
>(
  input: 
    | (theme: MantineTheme, params: Params, createRef: (refName: string) => string) => Input
    | Input
): (
  params: Params, 
  options?: UseStylesOptions<Key>
) => {
  classes: { [key in keyof Input]: string; };
  cx: (...args: any) => string;
  theme: MantineTheme;
};
```

Reply:
I think you're right. In that case I have a small preference to rename `UnThemedBlockNoteView` to something else (`InnerBlockNoteView`)? While I get what you mean, "UnThemed" sounds to me like it's not themed, which is not really the case